### PR TITLE
Box-drawing_character & Block_Elements

### PR DESCRIPTION
**Pull request by bluelovers**
+			// https://en.wikipedia.org/wiki/Box-drawing_character
+			(0x2500 <= x && x <= 0x257f) ||
+			// https://en.wikipedia.org/wiki/Block_Elements
+			(0x2580 <= x && x <= 0x259f) ||

### CODE REVIEW

**Comment by sindresorhus** [comment]
Don't do unrelated changes.

**Comment by bluelovers** [comment]
this fix work for cli-table2's table border

cos "─" is 2, not 1

before
---------------

┌─────────┬───────┬────────┬────────────────┐
│ address │ value │ active │ desc           │
├─────────┼───────┼────────┼────────────────┤
│ 7FEDFE  │ FF    │ false  │ 人物1 EXP 無限 │
├─────────┼───────┼────────┼────────────────┤
│ 7FF0FF  │ 90    │ false  │ p1 exp.1       │
├─────────┼───────┼────────┼────────────────┤
│ 7FF100  │ 02    │ false  │ p1 exp.2       │
├─────────┼───────┼────────┼────────────────┤
│ 7FF3FF  │ 90    │ false  │ p2 exp.1       │

after
---------------

┌─────┬────┬────┬────────┐
│ address  │ value  │ active │ desc           │
├─────┼────┼────┼────────┤
│ 7FEDFE   │ FF     │ false  │ 人物1 EXP 無限 │
├─────┼────┼────┼────────┤
│ 7FF0FF   │ 90     │ false  │ p1 exp.1       │
├─────┼────┼────┼────────┤
│ 7FF100   │ 02     │ false  │ p1 exp.2       │
├─────┼────┼────┼────────┤
│ 7FF3FF   │ 90     │ false  │ p2 exp.1       │
├─────┼────┼────┼────────┤

**Comment by sindresorhus** [comment]
Please look at the diff.

**Comment by bluelovers** [comment]
u mean `(0x20000 <= x && x <= 0x3fffd)` ?
this didn't make my change useless

----

```
const stringWidth = require('string-width');
console.log(stringWidth('─')); // before patch is 1, after is 2
```

File: index.js
Comment (sindresorhus) [inline index.js]: This is an unrelated change and should be removed.

File: index.js
Comment (sindresorhus) [inline index.js]: This is an unrelated change and should be removed.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by bluelovers** [comment]
just tell me is code style not match next time, not the code is wrong lol

https://github.com/bluelovers/is-fullwidth-code-point/commit/939af8060552955bb04026eeb56f311ecd22e552

**Comment by sindresorhus** [comment]
> just tell me is code style not match next time, not the code is wrong lol

I hope the next time you'll just follow the existing code style instead of wasting the time of the maintainer with silly things like this ;)

**Comment by sindresorhus** [comment]
Can you comment some source/proof here that the unicode ranges you specified are actually fullwidth?

**Comment by bluelovers** [comment]
code style is ide auto change it lol
idk how proof, but maybe u can see the url on the commit's comments

**Comment by sindresorhus** [comment]
Sorry, that's not good enough. I'm not going to merge this unless I have proof that this change actually does what you say it does. It's up to you as a submitter to provide such proof.

**Comment by bluelovers** [comment]
if this change patch on string-width
u will be ok or not?

**Comment by sindresorhus** [comment]
> if this change patch on string-width

Can you please clarify? I don't understand what you're asking.

**Comment by sindresorhus** [comment]
Closing for lack of response, test, or any kind of proof that this change is correct.

**Pull request closed without merging**
